docs/docs/developers/architecture/aws-ecs/index.md

```diff
@@ -31,8 +31,4 @@ Amazon Elastic File System (Amazon EFS) is fully managed, serverless file system
 ### Amazon Application Load Balancer
 Application Load Balancer handles all request from both site visitors (HTTPS on port `443`) and authors (HTTPS on port `8443`), and forward to `nginx` or `cms` ECS services. ALB also terminates HTTPS - all routing between ALB and ECS use HTTP (`80` / `8080`) because the connection is private. Read more details [here](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html).
 
-## Docker Compose ECS integration
```

Comment:
We lost this two points.

Reply:
they are moved to deployment guide but we agreed with Maciej to remove headers